iOS internship/ForecastModel.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ForecastModel.swift
+//  iOS internship
+//
+//  Created by pavels.vetlugins on 26/09/2018.
+//  Copyright © 2018 pavels.vetlugins. All rights reserved.
+//
+
+import Foundation
+import Freddy
+
+public class ForecastModel: FetchForecastDelegate{
+    
+    var delegate: GetForecastDelegate?
```

Comment:
Naming is a bit off here, delegate naming should reflect "Delegate of what thing" not "What this delegate does". So `ForecastDelegate` not `GetForecastDelegate`.